src/extracts/src/process_watcher.rs

```diff
@@ -19,14 +15,19 @@ use std::collections::HashMap;
 use std::collections::{hash_map::Entry::Vacant, HashSet};
 use std::path::{Path, PathBuf};
 use std::time::Duration;
-use sysinfo::ProcessStatus;
-use sysinfo::{Pid, Process, System};
+use sysinfo::{Pid, Process, ProcessStatus, System};
+use tracer_common::event::attributes::process::{
+    CompletedProcess, DataSetsProcessed, InputFile, ProcessProperties,
+};
+use tracer_common::event::attributes::EventAttributes;
+use tracer_common::recorder::EventRecorder;
+use tracer_common::target_process::{Target, TargetMatchable};
 
 pub struct ProcessWatcher {
     targets: Vec<Target>,
     seen: HashMap<Pid, Proc>,
     process_tree: HashMap<Pid, ProcessTreeNode>,
-    // We wanna track unique datasamples we come across when monitoring process args
+    // We want to track unique data samples we come across when monitoring process args
     datasamples_tracker: HashSet<String>,
```

Comment:
TO CREATE ANOTHER TASK: i don't know if tracking by file name is correct, need to review this on another task, but i think we should check the the file content hashing instead of the filename, we could have 2 different files with the same name, right? @davincios